classes/options-general.php

```diff
@@ -147,7 +147,7 @@ function print_page() {
 
 			if ( $count >= 250 ) {
 				$disabled_autoload = false; // let it happen. we're limiting to 500 in the query
-				$disabledmsg       = sprintf( __( 'You have too many locations to auto-load them all. Only the closest %d will be displayed if auto-load all is selected.', 'simplemap' ), '250' );
+				$disabledmsg       = sprintf( __( 'You have too many locations to auto-load them all. Only the closest %d will be displayed if auto-load all is selected.', 'SimpleMap' ), '250' );
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 195 characters

<!-- Comment by [SideCI](https://sideci.com) -->